<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1317</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Point location in 3D subdivision</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QWidget" name="mp_main_widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4"/>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="mp_list_renderables">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="mp_btn_load_mesh">
         <property name="text">
          <string>Load Mesh</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mp_btn_add_point">
         <property name="text">
          <string>Add Point</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mp_btn_remove_selected">
         <property name="text">
          <string>Remove Selected</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mp_btn_toggle_visibility">
         <property name="text">
          <string>Hide Selected</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mp_btn_view_all">
         <property name="text">
          <string>View All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mp_btn_view_selected">
         <property name="text">
          <string>View Selected</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mp_btn_toggle_view_mode">
         <property name="text">
          <string>Toggle View Mode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mp_btn_transform">
         <property name="text">
          <string>Transform</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mp_btn_subdivide">
         <property name="text">
          <string>Subdivide</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mp_btn_localize">
         <property name="text">
          <string>Localize</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
